core/src/test/scala/org/apache/spark/util/SslTestUtils.scala

```diff
@@ -15,8 +15,9 @@
  * limitations under the License.
  */
 
-package org.apache.spark
+package org.apache.spark.util
```

Comment:
cc @dongjoon-hyun FYI

Moving this file to the `core/src/test/scala/org/apache/spark/` directory is a solution that involves fewer changes. If you prefer to move the file instead of fixing the package name, please let me know.

Reply:
This looks like an incorrect package name - keeping it in `util` makes more sense.
Thanks for fixing this @LuciferYang !